packages/eslint-plugin/package.json

```diff
@@ -29,7 +29,6 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "fs-extra": "^8.1.0",
     "minimatch": "^3.0.4",
-    "@nrwl/tao": "13.8.1",
```

Comment:
this was causing having dupes in our dependencies. why was this added ? @TristanWatanabe

Reply:
added for typing purposes: https://github.com/microsoft/fluentui/blob/9d99c66f0a0fedc1757ea44accc09f9c07d5845a/packages/eslint-plugin/src/utils/configHelpers.js#L10